link to configuring plugin page from all plugin page (#1624)
facebookresearch/hydra: website/docs/plugins/ax_sweeper.md

  is_noisy: true
  params: {}
```

> You have multiple options configuring the Plugin, check out [this page](../patterns/configuring_plugins) for more info.

Reviewer: ```suggestion
There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more info.
```

  is_noisy: true
  params: {}
```

> There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more info.

Reviewer: ```suggestion
There are several standard approaches for configuring plugins. Check [this page](../patterns/configuring_plugins) for more information.
```

Sorry :)

Author: 😆 fixed!
